src/components/v5/frame/ColonyHome/partials/ReputationChart/utils.ts

```diff
@@ -107,16 +120,29 @@ export const getTeamReputationChartData = (
 export const getContributorReputationChartData = (
   contributorsList: ContributorItem[],
 ): ReputationChartDataItem[] => {
-  const topContributors = contributorsList
+  let topContributors = contributorsList
     .slice(0, WIDGET_TEAM_LIMIT)
     .map(({ walletAddress, user, reputation }, index) => {
       return {
         id: walletAddress,
         label: user?.profile?.displayName || '',
-        value: Number(reputation),
+        value: reputation || 0,
         color: getTeamHexColor(CONTRIBUTORS_COLORS_LIST[index]),
       };
-    });
+    })
+    .filter(({ value }) => value > 0);
+
+  if (topContributors.length <= WIDGET_TEAM_LIMIT) {
+    const adjustedValues = adjustPercentagesTo100(
+      topContributors.map((contributor) => contributor.value),
+      // getNormalisedDomainReputationPercentage rounds to two decimals
+      2,
+    );
+    topContributors = topContributors.map((contributor, idx) => ({
+      ...contributor,
+      value: adjustedValues[idx],
+    }));
+  }
```

Comment:
The problem with this condition is that the `topContributors` always has a max length of `WIDGET_TEAM_LIMIT`, so we can end up normalising the values for the `topContributors`, but there is still reputation remaining in the remaining contributors, leading to this situation where the values add up to `104.33%`

![Screenshot 2024-09-24 at 10 29 40](https://github.com/user-attachments/assets/f892ecaa-783f-4c6d-a24e-0c24f85c6166)

So, maybe it will make sense to apply `adjustPercentagesTo100` to the entire returned array

Reply:
Ah yeah, good point, I'll just apply that to the whole array then.

Reply:
Ah, actually, it was just a typo. I meant to use `contributorsList.length`